src/main/java/com/jagrosh/jmusicbot/commands/dj/MoveTrackCmd.java

```diff
@@ -0,0 +1,92 @@
+package com.jagrosh.jmusicbot.commands.dj;
+
+
+import com.jagrosh.jdautilities.command.CommandEvent;
+import com.jagrosh.jmusicbot.Bot;
+import com.jagrosh.jmusicbot.audio.AudioHandler;
+import com.jagrosh.jmusicbot.audio.QueuedTrack;
+import com.jagrosh.jmusicbot.commands.DJCommand;
+import com.jagrosh.jmusicbot.queue.FairQueue;
+
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+/**
+ * Command that provides users the ability to move a track in the playlist.
+ */
+public class MoveTrackCmd extends DJCommand
+{
+
+    private static final Pattern PATTERN = Pattern.compile("^(\\d+)\\s+(\\d+)$");
+
+    public MoveTrackCmd(Bot bot)
+    {
+        super(bot);
+        this.name = "movetrack";
+        this.help = "Move a track in the current playlist to a different position";
+        this.arguments = "<from> <to>";
+        this.aliases = new String[]{"move"};
+        this.bePlaying = true;
+    }
+
+    @Override
+    public void doCommand(CommandEvent event)
+    {
+        int from;
+        int to;
+
+        try
+        {
+            // Validate the args
+            String args = event.getArgs().trim();
+            Matcher matcher = PATTERN.matcher(args);
+            if (!matcher.matches())
+            {
+                event.replyError("That ain't right. Usage: movetrack <from> <to>");
+                return;
+            }
+
+            from = Integer.parseInt(matcher.group(1));
+            to = Integer.parseInt(matcher.group(2));
+        }
+        catch (NumberFormatException e)
+        {
+            // Should already be caught by the regex but ok..
+            event.replyError("That ain't a number: " + e.getMessage());
```

Comment:
same thing as above (language/tone), and also the regex wouldn't catch extremely large numbers that can't be parsed by integer